felucca/common/task.py

```diff
@@ -0,0 +1,67 @@
+class Task(object):
+    """Task Object
+    Task object represent a pharos executable task 
+    """
+
+    def __init__(self, task_executable_file, tool_type, command_line_input):
+        self.task_executable_file = task_executable_file
```

Comment:
Why task_executable_file is public field?

Reply:
Solved.